markdown/chapters/meandeck/core-concepts.md

```diff
@@ -0,0 +1,194 @@
+you are happy in what you do that's great. (Unless you want to play Delver in
+which case boooo!!.....just kidding). This is also the case with undertaking any
+activity. If you are not positively benefitting from it then it may be worth
+taking a break/step back and just having a chill from it for a bit.
+
+## Syntax Used
+
+### You Keep Saying *Pile*, What Does That Mean?
+
+The term *pile* is used to denote the stack of 5 cards with which you build your
+deck on successfully resolving :card[Doomsday]. A large portion of this site is
+to help you learn how to construct these *piles* in order to win the game in 
+almost any circumstance/matchup. While it would be nigh impossible to enumerate
+every game state and pile variation, these chapters are aimed to show you the
+basic concepts behind casting and winning with Doomsday. 
+
+### Pile Notation
+
+Many of the chapters will showcase a number of *piles*, their requirements and 
+the instructions to achieve them. For each one we shall present any mana 
+requirements or card requirements followed by the pile as a visual display.
+
+If we take the following example:
+
+> {U}{U} + Cantrip
+
+:::row{variant=PILE}
+- IU
+- LP
+- LP
+- TO
+- TO
+:::
+
+The cost omits that we require {B}{B}{B} for casting Doomsday as this is an
+assumed requirement for even considering what *pile* to build. It requires 
+{U}{U} for casting specific cards in the pile and the ability to *cantrip*
+(draw a card). If the chosen *cantrip* effect costs mana you will need to add 
+this on to the total otherwise presented. In the example of using 
+:card[Ponder] as your cantrip you will need {U}{U} + {U}.
+
+> {U} + Brainstorm + X
+
+Sometimes we will use the terminology of *X*. In this instance X means having
+any single card in hand, irrespective of what that card is. An X card is
+normally used in conjunction with :card[Brainstorm] and thus is rarely cast but
+we will explore more of this in another chapter.
+
+The instructions for executing a *pile* will be included below it. There may be 
+a paragraph or statement exploring some of the ideas within the *pile* but
+there should also be a step by step guide in how to execute the *pile too*.
+
+Let's return to the above example to see how this might be presented:
+
+ > {U}{U} + Cantrip
+
+:::row{variant=PILE}
+- IU
+- LP
+- LP
+- TO
+- TO
+:::
+
+This is a simple pile that uses :card[Ideas Unbound] to draw into 2 mana and
+:card[Thassa's Oracle]. It does not play around any interaction the opponent may
+have.
+
+*Pile* steps:
+
+1. Use your cantrip to draw into the pile, draw Ideas Unbound
+1. Cast Ideas Unbound, draw Lotus Petal x2 and Thassa's Oracle
+1. Cast both Lotus Petals and crack both for {U}
+1. Cast Thassa's Oracle and win the game with 1 card left in your deck
+
+You'll notice the term *crack* here. *Crack is a colloquial term for activating 
+a mana rock like LED or Lotus Petal or for sacrificing a Fetchland like
+:card[Polluted Delta].*
```

Comment:
Entire line "Crack is a colloquial term for activating 
 a mana rock like LED or Lotus Petal or for sacrificing a Fetchland like
 :card[Polluted Delta]." is currently italicized. Is this intentional or should it just be "Crack"?